themes/bbs/sass/ding/panel.scss

```diff
@@ -0,0 +1,220 @@
+//
+// Styling of panel layouts
+
+@import '../base.scss';
+
+// Layout
+//
+// Styleguide 4.
+
+//
+// Extra facetbrowsers
+// @TODO should these be removed from the panel?
+.primary-content .pane-ding-facetbrowser,
+.primary-content .pane-search-backends {
+  display: none;
+}
+
+// ==========================================================================
+// Panel Layouts
+// ==========================================================================
+
+.content-wrapper {
+  .pane-content {
+
+    // Ding frontpage layout
+    //
+    // Markup:
+    // <div class="panel-pane">
+    //  <div class="pane-content">
+    //    <div class="frontpage-layout">
+    //      <div class="panel-pane">
+    //        <div class="panel-pane-inner">
+    //          Panel
+    //          <div class="dummy-content"></div>
+    //        </div>
+    //      </div>
+    //      <div class="panel-pane">
+    //        <div class="panel-pane-inner">
+    //          Panel
+    //          <div class="dummy-content"></div>
+    //        </div>
+    //      </div>
+    //    </div>
+    //  </div>
+    // </div>
+    //
+    // Styleguide 4.1
+
+    .frontpage-layout {
+      .panel-pane {
+        @include linear-gradient(to top, $white 30px, $grey-light 100%);
+        // Has dynamic background
+        .has-dynamic-background & {
+          max-width: $dynamic-background-width;
+          margin: 0 auto;
+        }
+        .panel-pane-inner {
+          @include wrapper;
+          position: relative; // Is set to relative to position absolute elements inside panels correct
+          padding: 40px 0 30px;
+        }
+      }
+    }
+
+    // Ding2 Site template
+    //
+    // Markup:
+    // <div class="panel-pane">
+    //  <div class="pane-content">
+    //    <div class="ding2-site-template">
+    //      <div class="panel-pane">
+    //        <div class="panel-pane-inner">
+    //          Panel
+    //          <div class="dummy-content"></div>
+    //        </div>
+    //      </div>
+    //      <div class="panel-pane">
+    //        <div class="panel-pane-inner">
+    //          Panel
+    //          <div class="dummy-content"></div>
+    //        </div>
+    //      </div>
+    //    </div>
+    //  </div>
+    // </div>
+    //
+    // Styleguide 4.2
+
+    .ding2-site-template {
+      .panel-pane {
+        @include linear-gradient(to top, $white 30px, $grey-light 100%);
+        &.no-gradient {
+          background-image: none;
+          .panel-pane-inner {
+            padding-top: 20px;
+          }
+        }
+        // Has dynamic background
+        .has-dynamic-background & {
+          max-width: $dynamic-background-width;
+          margin: 0 auto;
+        }
+        .panel-pane-inner {
+          @include wrapper;
+          position: relative; // Is set to relative to position absolute elements inside panels correct
+          padding: 40px 0 30px;
+        }
+      }
+    }
+
+    // Default panel layout
```

Comment:
Why all this? It's possible to look in the ding2 template to see the original markup. :)